pom.xml

```diff
@@ -790,7 +790,7 @@
                 <skipIntegrationTests>true</skipIntegrationTests>
                 <!-- Once we truly run tests with Testcontainers, this should be switch to "docker", activating ITs -->
                 <packaging.type>docker-build</packaging.type>
-                <postgresql.server.version>13</postgresql.server.version>
+                <postgresql.server.version>16</postgresql.server.version>
```

Comment:
@donsizemore could you please add a release note snippet?

I believe the [docs](https://guides.dataverse.org/en/6.1/installation/prerequisites.html#postgresql) are ok, for now. They say, "PostgreSQL 13 is recommended because it’s the version we test against." And that's what we test with in Jenkins, version 13: https://github.com/GlobalDataverseCommunityConsortium/dataverse-ansible/blob/3cfe7e1c07b47e7e1514d1145bd64d52c08f4680/tests/group_vars/jenkins.yml#L272

Reply:
Don added a release note snippet, thanks!